<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>707</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>978</width>
    <height>707</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Medical Image Reconstruction</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Original Image</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="1">
        <widget class="QLabel" name="imageLabel">
         <property name="maximumSize">
          <size>
           <width>453</width>
           <height>277</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Reconstruction</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="reconsLayout"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Sinogram</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="sinogramLayout"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Medical Image Reconstruction</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>471</width>
         <height>281</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0">
         <widget class="QLabel" name="projectionLabel">
          <property name="text">
           <string>Number of Projections:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QComboBox" name="filterComboBox">
          <item>
           <property name="text">
            <string>Filtered Back Projection</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SART</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="filterLabel">
          <property name="text">
           <string>Filter Type:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="projectionValue">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="calculateButton">
          <property name="text">
           <string>Calculate</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="methodComboBox">
          <item>
           <property name="text">
            <string>ramp</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>shepp-logan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>cosine</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>hamming</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>hann</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="clearButton">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="methodLabel">
          <property name="text">
           <string>Reconstruction Method:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="maxLabel">
          <property name="text">
           <string>Maximum Angle:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="animationButton">
          <property name="text">
           <string>Animation</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="angleValue"/>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="loadButton">
          <property name="text">
           <string>Load Image</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>978</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
